src/platform/site-wide/user-nav/tests/e2e/00-required.cypress.spec.js

```diff
@@ -20,6 +20,23 @@ const mockFeatureToggles = () => {
     },
   });
 };
+const enableDropdownComponent = () => {
+  cy.route({
```

Comment:
cy.route has been deprecated in favor of cy.intercept. See https://docs.cypress.io/api/commands/route for more info.